src/libraries/System.IO.MemoryMappedFiles/tests/MemoryMappedFile.CreateFromFile.Tests.cs

```diff
@@ -768,6 +769,7 @@ public void LeaveOpenRespected_OutstandingViews(bool leaveOpen)
         /// Test to validate we can create multiple maps from the same FileStream.
         /// </summary>
         [Fact]
+        [PlatformSpecific(~TestPlatforms.Browser)] // the emscripten implementation doesn't share data
```

Comment:
What does it mean? Is the behaviour unexpected?

Reply:
@marek-safar kind of, though sharing an mmap'd file is not the common case. we could probably looking into detecting the situation and throwing a proper exception (right now the second mmap just doesn't see the modifications from the first mmap)

Reply:
Please create a tracking issue instead of ignoring the tests

Reply:
https://github.com/dotnet/runtime/issues/39428